scripts/history.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const HISTORY_LIMIT = 20;
+
+export class History {
+
+    constructor(key, onHistoryChange) {
```

Comment:
How about `onHistoryUpdate`?
```suggestion
    constructor(key, onHistoryUpdate) {
```

Reply:
Changed it.
https://github.com/dhruvdhaduk-simform/scientific-calc/pull/5/commits/6b7fa988e45cac5b2225efde074b7d790214b01f